src/connector/src/parser/json_parser.rs

```diff
@@ -80,28 +80,26 @@ impl JsonParser {
     #[allow(clippy::unused_async)]
     pub async fn parse_inner(
         &self,
-        payload: &[u8],
+        payload: Vec<u8>,
```

Comment:
I mean as long as it is `&[u8]`, there is no way for us to pass a `&mut [u8]` to `simd_json::to_borrowed_value`, right?

Reply:
I think so.